<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="quit_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
	background-color: transparent; 
	border: 0px 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="start_game_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.5, fy: -0.4,
        radius: 3, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.1, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.5, fy: -0.1,
        radius: 2, stop: 0 #fff, stop: 1 #ddd
        );
    }

</string>
     </property>
     <property name="text">
      <string>Начать игру</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="restart_button">
     <property name="font">
      <font>
       <family>Berlin Sans FB Demi</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.5, fy: -0.4,
        radius: 3, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.1, stop: 0 #fff, stop: 1 #bbb
        );
	padding: 5px;
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.5, fy: -0.1,
        radius: 2, stop: 0 #fff, stop: 1 #ddd
        );
	padding: 5px;
    }</string>
     </property>
     <property name="text">
      <string>Сгенирировать слова</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QScrollArea" name="hints_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>420</width>
        <height>598</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="settings_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
	background-color: transparent; 
	border: 0px 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>hints_scroll_area</zorder>
  <zorder>start_game_button</zorder>
  <zorder>restart_button</zorder>
  <zorder>settings_button</zorder>
  <zorder>quit_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
